### ISSUE TITLE
Login Page UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#28]
## User story
1. As a user
2. I want / need to see a login section
3. So that I can enter my login data 

## Acceptance criteria
* Login section appears after hitting the login button
* Login section provides fields to enter the login data
* Login section contains a button to start the login function

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
Once the overview page exists, the user should see that after logging in - doesn't have to be implemented here
